### PR TITLE
Add a Vulnerability Disclosure and Embargo Policy

### PR DESCRIPTION
**Pull request by nickvidal**
Change status of RFC 00002-vulnerability-disclosure-and-embargo-policy to adopted. The CCC has requested that all CCC projects have a disclosure policy.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+The Enarx project welcomes the responsible disclosure of vulnerabilities, including those discovered in:
+- Enarx code
+- Enarx dependencies (even compilers)
+- Enarx documentation
+- 3rd party hardware or firmware
+
+## Initial Contact
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.dev or on the
```
Comment (dcmiddle) [inline SECURITY.md:12]: Is security@enarx.dev monitored? I sent a test note to it 2021-09-15 and did not get a reply (though my company email filters could have interfered).

**Comment by dcmiddle** [comment]
I believe this policy satisfies CCC requirements. See https://github.com/confidential-computing/governance/blob/master/security-response-policies.md

File: SECURITY.md (hunk 1 above)
Comment (nickvidal) [reply SECURITY.md:12]: Hi @dcmiddle, I'll check if this email is getting monitored. Thanks!

Comment (nickvidal) [reply SECURITY.md:12]: Thanks Dan for testing. Ashley has solved this now. The emails were being sent to Mike and Nathaniel's RH address.

**Comment by npmccallum** [comment]
@nickvidal This PR no longer matches against the `main` branch. It needs to be reworked against the new `main`.

**Pull request closed without merging**